source/B2CWebApi/Models/RequestWholesaleServicesMarketDocument.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Energinet.DataHub.EDI.B2CWebApi.Models;
+
+/// <summary>
+///     Responsible for carrying the market message data from the incoming message before any data validation.
+/// </summary>
+public sealed record RequestWholesaleSettlementMarketRequest(
+    CalculationType CalculationType,
+    string StartDate,
+    string EndDate,
```

Comment:
Is EndDate not nullable or can't not be null from the UI?
```suggestion
    string? EndDate,
```
It looks like RequestWholesaleSettlementDtoFactory is treating it as nullable

Reply:
The same applies to aggregations iirc. I made them "identical" to make things easier to understand. I'm tempted to keep it as-is to avoid having to rerun the things in BFF and to keep it consistent. Ideally, I would prefer if we changed it in such a way that we skip the internal DTO and just translate to the B2B one.